pyfolio/plotting.py

```diff
@@ -611,10 +611,18 @@ def plot_rolling_returns(
             label='Live', ax=ax, **kwargs)
 
         if cone_std is not None:
-            cone_df = timeseries.cone_rolling(
-                returns,
-                num_stdev=cone_std,
-                cone_fit_end_date=live_start_date)
+            # check to see if we're just rendering a single cone, or multiple
+            # cones at various stddevs, defined as elements of cone_std
+            if type(cone_std) is list:
```

Comment:
Instead of having separate logic for list and float case. I would convert every float to a list, then have one loop.